### ISSUE TITLE
test_bitcoin-qt failure

### ISSUE THREAD

**Issue by lexical** [dogecoin/dogecoin#1228]
1.10-dev HEAD:
```
=============================================
   Bitcoin Core 0.11.0: src/test-suite.log
=============================================

# TOTAL: 2
# PASS:  1
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: qt/test/test_bitcoin-qt
=============================

********* Start testing of URITests *********
Config: Using QtTest library 5.2.1, Qt 5.2.1
PASS   : URITests::initTestCase()
PASS   : URITests::uriTests()
PASS   : URITests::cleanupTestCase()
Totals: 3 passed, 0 failed, 0 skipped
********* Finished testing of URITests *********
********* Start testing of PaymentServerTests *********
Config: Using QtTest library 5.2.1, Qt 5.2.1
PASS   : PaymentServerTests::initTestCase()
QDEBUG : PaymentServerTests::paymentServerTests() PaymentServer::initNetManager: No active proxy server found.
FAIL!  : PaymentServerTests::paymentServerTests() Compared values are not the same
   Actual   (merchant)                   :
   Expected (QString("testmerchant.org")): testmerchant.org
   Loc: [qt/test/paymentservertests.cpp(87)]
PASS   : PaymentServerTests::cleanupTestCase()
Totals: 2 passed, 1 failed, 0 skipped
********* Finished testing of PaymentServerTests *********
```

**Comment by lexical** [comment]
PASS: test/test_bitcoin
FAIL: qt/test/test_bitcoin-qt
make[6]: Entering directory `/home/kengyu/git/dogecoin/src'
make[7]: Entering directory `/home/kengyu/git/dogecoin/src'
make[7]: Leaving directory `/home/kengyu/git/dogecoin/src'
make[6]: Leaving directory `/home/kengyu/git/dogecoin/src'
============================================================================
Testsuite summary for Bitcoin Core 0.11.0
============================================================================
# TOTAL: 2
# PASS:  1
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0
============================================================================
See src/test-suite.log
Please report to https://github.com/bitcoin/bitcoin/issues
============================================================================

**Comment by patricklodder** [comment]
We're aware of this issue, it is caused by payment protocol fixtures that need to be regenerated. Thank you for reporting it.

**Comment by rnicoll** [comment]
Assigning to myself as I probably should look at this. Likely to be Mon/Tues though, sorry.

**Comment by patricklodder** [comment]
@rnicoll the fixtures in question are these: https://github.com/dogecoin/dogecoin/blob/1.10-dev/src/qt/test/paymentrequestdata.h#L57-L460

**Comment by langerhans** [comment]
Put the logs into ``` blocks to fix formatting. Not actually sure why those tests would break, even before we changed the payment request handling.  
Not sure if this is still current, but it might come in handy: https://github.com/gavinandresen/paymentrequest

**Comment by patricklodder** [comment]
I'll look into this and try to have this patched for 1.10.1

**Comment by patricklodder** [comment]
fixed with #1341
